versions.tf

```diff
@@ -1,14 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.0.0"
```

Comment:
Woah. This is the first time we're doing this I believe.

Reply:
We still have engagements using older versions. I think we'll need another approval on this if this is needed. Could you explain why this is being bumped to 1.0 ?

Reply:
There's an issue with https://github.com/cloudposse/terraform-aws-s3-bucket/blob/master/main.tf#L376 not being able to be determined when the module enabled=false

But I see

Reply:
I did explain it briefly in the _why_ section of my PR description

> The comprehensive Terratest suite in examples/complete and test/src will only work with Terraform 1.0.0 and above, and otherwise will result in "count cannot be determined before apply" errors.